ebos/equil/initstateequil.hh

```diff
@@ -669,13 +669,20 @@ equil_GOW(const Region& reg, const VSpan& span)
         this->makeOilPressure(ic, reg, span);
     }
 
-    if (this->waterActive()) {
+    if (this->waterActive() && this->oilActive()) {
         // Pcow = Po - Pw => Pw = Po - Pcow
         const auto ic = typename WPress::InitCond {
             reg.zwoc(),
             this->oil(reg.zwoc()) - reg.pcowWoc()
         };
 
+        this->makeWatPressure(ic, reg, span);
+    } else if (this->waterActive() && !this->oilActive()) {
+        // No oil phase set Pw = Pg - (Pcgo + Pcow)
+        const auto ic = typename WPress::InitCond {
+            reg.zgoc(),
+            this->gas(reg.zgoc() - (reg.pcgoGoc() + reg.pcowWoc()))
```

Comment:
Oops, this should probably say
```C++
this->gas(reg.zgoc()) - (reg.pcGoc() + reg.pcowWoc());
```
instead.  Note closing parenthesis on the `this->gas()` call.  I guess there's also an implied assumption that the gas/oil and oil/water contacts are at the same depths, but I'll need to check the details.

Reply:
Thanks for noticing. 

I will provide a similar fix for case when OWC/GOC < DATUM.

>I guess there's also an implied assumption that the gas/oil and oil/water contacts are at the same depths, but I'll need to check the details.

Cool. I have only testet with OWC=GOC.

Reply:
I checked the details and I was wrong before.

In the gas/water case then item 3 of the `EQUIL` keyword is the GWC and item 4 is the gas/water capillary pressure (Pcgw).  In all other cases item 3 is OWC and item 4 is the Pcow.  In other words we should really write
```C++
const auto ic = typename WPress::InitCond {
    reg.zwoc(), // The WOC is really the GWC for gas/water cases
    this->gas(reg.zwoc()) - reg.pcowWoc() // Pcow(WOC) is really Pcgw(GWC) for gas/water cases
};
```
here.  Sorry for adding to the confusion.

On the other hand we may need the `Pcwg = Pcgo + Pcow` relation for when we invert the capillary pressure relations to derive initial saturations.

Reply:
Thanks for digging in to this. Seems to work as expected for both  WGC < DATUM and WGC > DATUM. Note the test case is without capillary pressure.